soc/xtensa/nxp_adsp/rt6xx/Kconfig.soc

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2023 Google LLC.
+# Copyright (c) 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "NXP RT6xx ADSP SoC Selection"
+	depends on SOC_SERIES_NXP_RT6XX
+
+	config SOC_NXP_RT685
+		bool "NXP RT685"
+		select HAS_MCUX
+		select HAS_MCUX_FLEXCOMM
```

Comment:
We don't need to select these Kconfigs to enable the drivers, that is now driven by devicetree. SOCs that select this currently just do so for legacy reasons (and can be cleaned up)